docs/manual-curation.md

```diff
@@ -6,8 +6,6 @@ The goal is for traits with occurence ≥ 10 to have 100% coverage after the man
 At this step, mappings produced by the pipeline on the previous iteration (including automated and manual) are downloaded to be used to aid the manual curation process.
 
 ```bash
-# Set the variable below for year and month of the OpenTargets batch release
-export BATCH_ROOT=/nfs/production3/eva/opentargets/batch-YYYY-MM
```

Comment:
why delete this? it's needed for the very next command

Reply:
All variables are now configured in one place, which is described in the beginning of the main protocol. Added clarification: https://github.com/EBIvariation/eva-cttv-pipeline/pull/80/commits/398f7dd487612b4b42b690ced112cb6642044712